src/main/java/codeu/model/store/basic/MessageStore.java

```diff
@@ -106,4 +107,15 @@ public void addMessage(Message message) {
   public void setMessages(List<Message> messages) {
     this.messages = messages;
   }
+
+  /** Adds all messages by the specified user into a list **/
+  public List<Message> getMessagesByUser(User user) {
```

Comment:
Could you add tests for this in MessageStoreTest.java? Similar to testGetMessagesInConversation, which is already there.